src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/service/CourtCaseEventsServiceTest.kt

```diff
@@ -157,30 +151,4 @@ class CourtCaseEventsServiceTest {
     verify(personRecordService, never()).createNewPersonAndDefendant(person)
     verify(telemetryService).trackEvent(TelemetryEventType.HMCTS_PARTIAL_MATCH, mapOf("Surname" to "Jones"))
   }
-
```

Comment:
https://github.com/ministryofjustice/hmpps-person-record/blob/87dbc4ead36aecd403d565e5170aec75e4324bc9/src/test/kotlin/uk/gov/justice/digital/hmpps/personrecord/message/listeners/CourtCaseEventsListenerIntTest.kt#L55 covers this functionality